modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -45,14 +45,20 @@ The first task to adding an API to the unified schema is adding the API to Anypo
 * If your API is managed by API Manager or if you have added the API’s instances in Exchange, select *Get an existing URL from Anypoint Platform*.
 * If you prefer to manually provide your own implementation URL, select *Add a new URL*.
 . Click *Next: Provide authentication*.
-. On the *Provide authentication* page, select the type of authentication on the GET endpoints for the API that you are adding:
+. On the *Configure Security* page, select the type of authentication on the GET endpoints for the API that you are adding:
 +
 * *No Auth*: Use if your API is public.
 * *Basic Auth*: Use if you authenticate to your API using a username and password.
 * *Pass-through*: Use if you authenticate to your API by passing authorization headers.
 * *Client ID enforcement via headers*: Use if you authenticate to your API passing `client_id` and `client_secret` headers.
 * *Client ID enforcement via query parameters*: Use if you authenticate to your API passing `client_id` and `client_secret` query params.
 * *Custom*: Use if you authenticate to your API using custom header parameters and values.
+. Still on the *Configure Security* page, optionally configure mutual authentication (mTLS) between the API and DataGraph. 
+.. Select *Configure Keystore* and provide the following:
+* A public certificate signed by your organization's “root” certificate authority (CA)
+* The private key used to generate the certificate signing request (CSR)
+* The password for the encrypted private key
+.. Optionally, select *Configure Truststore* and provide a link to the root CA.
```

Comment:
the user will not configure mTLS as a whole, the keystore and the truststore are two separate containers that the user can optionally configure. So the user will be able to include either the keystore or the truststore, or both if needed. Also for the truststore, the user uploads files just as it would do for the keystore's public certificate (the user can upload one file with many certificates, or many files with one certificate, or a mix of both)